README.txt

```diff
@@ -1,26 +1,9 @@
-SOCIAL AUTH GOOGLE MODULE
```

Comment:
Please run PHPSniffer before any commit. You're not following the coding standard